modules/azure/frontdoor_test.go

```diff
@@ -0,0 +1,66 @@
+package azure
+
```

Comment:
Add a comment that Azure tests are tagged to be excluded from the CircleCI pipeline like this https://github.com/gruntwork-io/terratest/blob/e19192d2a19b27f52bb368bce54ccefa8fd8cd27/modules/azure/appService_test.go#L3